api/v1/handlers.go

```diff
@@ -548,17 +552,6 @@ func (s *apiV1) handleAdd(c echo.Context, u *util.User) error {
 
 	defer fi.Close()
 
-	replication := s.cfg.Replication
```

Comment:
pinOp code path doesn't support a custom replication setting, so removing this in favor of using what's configured by the content manager

Reply:
No, this endpoint allows users to submit a replication count, you can pass that into `PinContent`.  Pass default for the other places this is called